doc/develop/api/api_lifecycle.rst

```diff
@@ -236,9 +236,8 @@ The following are the requirements for deprecating an existing API:
   - Code using the deprecated API needs to be modified to remove usage of said
     API
   - The change needs to be atomic and bisectable
-  - Create a GitHub issue to track the removal of the deprecated API, and
-    add it to the roadmap targeting the appropriate release
-    (in the example above, 1.16).
+  - Add an entry in the `GitHub issue <https://github.com/zephyrproject-rtos/zephyr/issues/76943>`_
+    tracking removal of deprecated APIs.
```

Comment:
You mean that people edit the main post? If so, will everyone have the rights to do that?